DOCS/man/mpv.rst

```diff
@@ -666,7 +670,7 @@ Option                  Configuration file entry
 ``--flag``              ``flag``
 ``-opt val``            ``opt=val``
 ``--opt=val``           ``opt=val``
-``-opt "has spaces"``   ``opt="has spaces"``
+``-opt "has spaces"``   ``opt=has spaces``
```

Comment:
Had no idea this even worked...

Reply:
Me and avih had to reverse engineer the mpv.conf parsing code one year ago on IRC to discover how it really works, lol. It was high time we fixed the documentation. Actually another undocumented thing is that you can have `--` before option names in mpv.conf and it still works, but I don't think there is value in mentioning that.